src/methods/submitPaymentTransaction.ts

```diff
@@ -71,27 +98,28 @@ const buildPaymentTransaction = async ({
         amount: amount.toString(),
       });
     } else {
+      log.instruction({
+        title:
+          "Destination account does not exist, we are creating and funding it.",
+      });
+
       // If destination account is not funded, create and fund it
       operation = StellarSdk.Operation.createAccount({
         destination,
         startingBalance: amount.toString(),
       });
     }
 
-    // TODO: Do we need to add memo?
-
     transaction = new StellarSdk.TransactionBuilder(source, {
-      // TODO: Do we need to have a custom fee option?
       fee: BASE_FEE,
```

Comment:
We should probably be consistent with our base fee usage. I see some other instances of using 5X the base fee.

Reply:
We'll handle consistent `BASE_FEE` in https://github.com/stellar/nebula/issues/157.